client/src/components/Draftail/CommentableEditor/CommentableEditor.js

```diff
@@ -0,0 +1,217 @@
+import { DraftailEditor, ToolbarButton, createEditorStateFromRaw, serialiseEditorStateToRaw } from 'draftail';
+import { EditorState, Modifier, RichUtils, SelectionState } from 'draft-js';
+import { filterInlineStyles } from "draftjs-filters";
+import { useEffect, useMemo, useRef, useState } from 'react';
+import { useSelector, shallowEqual } from 'react-redux';
+
+import { STRINGS } from '../../../config/wagtailConfig';
+import Icon from '../../Icon/Icon';
+
+const COMMENT_STYLE_IDENTIFIER = 'COMMENT-'
+
+function usePrevious(value) {
+  const ref = useRef();
+  useEffect(() => {
+    ref.current = value;
+  }, [value]); // Only re-run if value changes
```

Comment:
```suggestion
  }, [value]);
```

On this very line I don’t think this comment is useful enough, but if you think it helps overall I’d suggest adding it as a JSDoc comment for the function.